content/doc/tutorials/tutorial-for-installing-jenkins-on-IBM-Cloud.adoc

```diff
@@ -5,202 +5,225 @@ section: doc
 ---
 
 :toc:
-:toclevels: 3
+:toclevels: 4
 :imagesdir: ../../book/resources/
 
-You should have an IBM Cloud account, otherwise you can http://cloud.ibm.com/registration[register here].
+You should have an IBM Cloud account, otherwise you can link:http://cloud.ibm.com/registration[register here].
 At the end of the tutorial you will have a cluster with Jenkins up and running.
 
-1. We will provision a new Kubernetes Cluster for you if, you already have one skip to <<Step 2 deploy IBM Cloud Block Storage plug-in,step **2**>>
-2. We will deploy  the <<Step 2 deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if already have it skip to <<Step 3 deploy Jenkins,step **3**>>
-3. <<Step 3 deploy Jenkins,Jenkins deployment>>
+. We will provision a new Kubernetes cluster for you, if you already have one skip to <<Deploy IBM Cloud Block Storage plug-in,step 2>>.
+. We will deploy  the <<Deploy IBM Cloud Block Storage plug-in,IBM Cloud Block Storage plug-in>>, if you already have it skip to <<Deploy Jenkins,step 3>>.
+. We will <<Deploy Jenkins,deploy Jenkins>>.
+. We will <<Verify Jenkins installation,verify jenkins installation>>.
```

Comment:
General comment: we should use present tense if possible and avoid future tense. In some cases, it's unavoidable, and that's fine. It's just something to keep in mind.

We can adjust the wording slightly to use present tense and update the jump links to match the suggestions below. (You may want to review the suggestions for these headings below first, to make sure they are acceptable before implementing these suggestions for the overview steps.)

```suggestion
. <<Provisioning a Kubernetes cluster,Provision a Kubernetes cluster>>. If you already have a Kubernetes cluster, you can skip to step 2.
. <<Deploying the IBM Cloud Block Storage plug-in,Deploy the IBM Cloud Block Storage plug-in>>. If you have already deployed the the IBM Cloud Block Storage plug-in, you can skip to step 3.
. <<Deploying Jenkins,Deploy Jenkins>>.
. <<Verifying the Jenkins installation,Verify the Jenkins installation>>.
```